### ISSUE TITLE
calendar horizontal placement broken

### ISSUE THREAD

**Issue by teleshoes** [taffybar/taffybar#455]
simpleclock calendar widget popup placement is incorrect when you first open it.
closing it and reopening fixes the position.

**To Reproduce**
click on clock, see calendar popup half off the screen

**Expected behavior**
calendar popup should fit on screen.

**Screenshots**
first time (calendar is cut off on right edge of screen):
![39851418-a9d08a88-53e4-11e8-918d-8390b0b55b66](https://user-images.githubusercontent.com/592062/58976748-3543f100-8796-11e9-983c-ebf3bffd329c.png)

subsequent times (calendar is aligned to right edge of screen):
![39851419-ae8f88da-53e4-11e8-9b7f-d4c48f3752bf](https://user-images.githubusercontent.com/592062/58976749-35dc8780-8796-11e9-983f-0fc45948d32c.png)

**Version information**
taffybar: 66763bca
Gtk version: 3.24.8 (debian libgtk-3-0 3.24.8-1)

**Comment by teleshoes** [comment]
see #261
@u11gh @IvanMalison

**Comment by colonelpanic8** [comment]
@teleshoes is this still an issue?

**Comment by teleshoes** [comment]
@colonelpanic8 yes, horizontal placement is still broken on first show in 7.2.4
vertical placement was fixed many years ago.

=> there is no change for me since this comment:
https://github.com/taffybar/taffybar/issues/261#issuecomment-499116186
